### DISCUSSION TITLE
Adding a tutorial "Simulation of a pendulum" to the docs.

### DISCUSSION THREAD

**Discussion by kolibril13** [ManimCommunity/manim#856] [Ideas]
Wuhu, we have now GitHub discussions, I am curious how this might be useful!

For the tutorials, I had the idea of writing a step by step tutorial (https://docs.manim.community/en/latest/tutorials.html) on how I would start to simulate a basic physics scene in manim,
including all the steps from choosing colours, defining formulas,  adding updaters, to thinking about rate functions.
And this could be done with a pendulum as example scene.

**Comment by eulertour** [comment]
Maybe it's just because I've used manim a lot but I think a pendulum alone would be a very short tutorial, mostly rotating a line + circle based on a cosine. Doing it with multiple periods simultaneously might cover more concepts and give a more visually appealing result.

**Comment by leotrs** [comment]
I'd rather keep the tutorials on our documentation strictly related to how to use the code as much as possible. For example, things like choosing colors and defining formulas are indeed important to create an effective animation but they are not directly related to the actual use of manim, or any functionality provided by it.

If you condense your idea down to the manim features it would be showcasing, then as @eulertour said, it would boil down to animating a couple of mobjects on a fairly simple path. Which is perhaps a bit too simple for a whole tutorial.

**Comment by kolibril13** [comment]
Thanks for your feedback!
Yep, this tutorial would not focus on the functionality of manim, but on the actual use of it.
It would narrate from the idea of having a physical problem that you want to transform into an animation.
As @eulertour said, only animating a pendulum would go quite fast, but I think there is so much more that one could illustrate in this example.
This would include how to structure the code so that one can easily change physical magnitudes (e.g. how can I quickly change the gravitational field without rewriting lots of code), or how I can make a class "pendulum" which easily allows multiple pendulums displayed on the screen.
If you think that the docs are not the right place to showcase these "use cases" of manim, I will probably make a blogpost on this idea at another place.

**Comment by kolibril13** [comment]
And here is what could be the result of the tutorial:
![Example](https://user-images.githubusercontent.com/44469195/102012743-76826d00-3d4c-11eb-9ee8-1ccfa4471b33.gif)

**Reply by SuperUser-Vermeylen** [reply]
I like this.
There should be a place dedicated to example scenes under 10 seconds (maybe even less).
A place where scenes have keywords associated with them, so we could all copy and paste scenes we need.
The current problem is the lack of keywords to easily wade through the exampl escenes in the galleries.
